modules/core/shared/src/main/scala/data/LTree.scala

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2018-2021 by Rob Norris
+// This software is licensed under the MIT License (MIT).
+// For more information see LICENSE or https://opensource.org/licenses/MIT
+
+package skunk.data
+
+import cats.Eq
+
+final case class LTree private (labels: List[String]) {
```

Comment:
This needs to be `sealed abstract case class` in order to suppress `copy` and companion `apply`.

Reply:
Thought this issue was fixed in `2.13.something` but good point.

Reply:
Yeah we still build for 2.12